### PR TITLE
First try to commit, EDAV CC

### PR DESCRIPTION
**Pull request by AbelPV**

### CODE REVIEW

File: Improved_density_estimation.Rmd [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+  
+}
+
+```
+
+## Examples
+
+Now we show some examples in which the function would be useful.
+
+### 1) Non-negative data with repetitions
```
Comment (Abhimanyu210100) [inline Improved_density_estimation.Rmd:277]: Please remove numbers after #

**Comment by Abhimanyu210100** [comment]
@AbelPV Please change your filename to contain only lower case letters (for eg. improved_density_estimation.Rmd)

**Comment by Abhimanyu210100** [comment]
Screenshot 2021-11-12 at 9 28 31 AM

The change is not being reflected in the filename. Could you do it again?

**Comment by Abhimanyu210100** [comment]
@AbelPV yes it is necessary to have the file name to be lowercased. @jtr13 could you please help us out with this issue? @AbelPV is unable to change the file name

**Comment by alexnsaunders** [comment]
You should be able to edit the filename fyi

https://docs.github.com/en/repositories/working-with-files/managing-files/renaming-a-file

Hope this helps.

**Comment by jtr13** [comment]
The problem is that git is case-insensitive for filenames but GitHub clearly isn't, at least in terms of how it sorts the files in the repo. Here's one option for dealing with this https://git.wtf/how-to-change-a-file-names-capitalization-without-changing-its-content  Another option is to change the name to `temp.Rmd` and then to the new lowercase name (https://stackoverflow.com/a/11183844/5314416) More on the issue: https://aqandrew.com/blog/git-and-case-sensitivity/
@alexnsaunders: I think this is the issue we were having with one of the chapters that had `.rmd` in `_bookdown.yml` but `.Rmd` as the actual file name (or vice versa).

**Comment by jtr13** [comment]
Ok, great. What method did you use?

**Comment by AbelPV** [comment]
I used the instructions in the first link, changing the file name in git with the comand

git mv --force Improved_density_estimation.Rmd improved_density_estimation.Rmd

**Comment by jtr13** [comment]
@Abhimanyu210100 Can you review and merge if it's ready to go?

**Comment by Abhimanyu210100** [comment]
@jtr13 Merged it, apologies for the delay. @AbelPV Thank you for your contribution 🎉

**Pull request merged**